src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import handlers.BalanceHandler;
+import handlers.OperationTypeHandler;
+import handlers.PurchaseHandler;
+import handlers.ReturnHandler;
+import handlers.SupplyHandler;
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.ReportService;
+import service.TransactionParserService;
+import serviceimpl.ReaderImpl;
+import serviceimpl.ReportImpl;
+import serviceimpl.TransactionParserImpl;
+import serviceimpl.WriterImpl;
+import strategy.StrategyChoosing;
+import strategy.StrategyImpl;
+
+public class Main {
+    private static final File INPUT_FILE = new File("src/main/java/resources/input.csv");
+    private static final File OUTPUT_FILE = new File("src/main/java/resources/output.csv");
+    private static ReaderImpl reader = new ReaderImpl();
+    private static WriterImpl writer = new WriterImpl();

```

Comment:
In main() method you should create a interface objects, not implementation objects